src/Tag/index.scss

```diff
@@ -0,0 +1,30 @@
+.nds-tag {
+  padding: 8px;
+  border-radius: 16px;
+  display: flex;
+  align-items: baseline;
+  font-size: 12px;
+  .narmi-icon-x {
+    color: RGB(var(--rgb-white));
+    cursor: pointer;
+    padding-left: 8px;
+    font-size: 12px;
+    position: relative;
+    top: 1px;
+  }
+}
+.nds-tag--dismissible {
+  color: RGB(var(--rgb-white));
```

Comment:
the `--color` variables don't need the RGB wrapper - you can just do:

```css
color: var(--color-white);
```

The `--rgb` variables are for when you need to transparentize a color by mixin in an alpha channel like `rgba(var(--rgb-pine), var(--alpha-20));`